packages/now-build-utils/src/prerender.ts

```diff
@@ -32,6 +40,22 @@ export class Prerender {
     }
     this.group = group;
 
+    if (bypassToken == null) {
```

Comment:
```suggestion
    if (!bypassToken) {
```

Ok I see now that you are covering both `null` and `undefined` here. What do you think of this change?

Reply:
This captures both `null` and `undefined`. Using only `!` will permit an empty string, which we do not want.

Reply:
If you check `typeof bypassToken === 'string'` and then `!bypassToken` after then it wont allow empty string :smart:

Reply:
Not sure I understand. The if clause would just be moving.